scripts/utils/render.js

```diff
@@ -27,19 +28,22 @@ const projectsCardBuilder = (projectsArr)=> {
 // projects form
 
 const overviewCardBuilder = (reposArray) => {
-  const pinnedRepos = reposArray.filter((r) => r.isPinned);
+  const pinnedRepos = reposArray
+    .map((r, index) => ({...r, index}))
```

Comment:
Here I add the index to the element so that I can reference it later on the unpin button